doc/host_config_schema/cchost_config.json

```diff
@@ -76,18 +76,81 @@
                 "properties": {
                   "authority": {
                     "type": "string",
-                    "enum": ["Node", "Service"],
+                    "enum": ["Node", "Service", "ACME"],
                     "default": "Service",
                     "description": "The type of endorsement for the TLS certificate used in client sessions. If the endorsement is not available, client sessions will be terminated, before the TLS handshake is complete. 'Node' means self-signed, 'Service' means service-endorsed."
                   }
                 },
                 "required": ["authority"],
                 "additionalProperties": false
+              },
+              "acme_configuration": {
```

Comment:
Shouldn't that be inside `"endorsement"`?

Reply:
That's just an enum, but in theory we could promote it to enum+name, yes.

Reply:
`"endorsement"` is an object with a single property `"authority"` (the enum). The idea was that any endorsement-related details go in as additional fields.

Reply:
Oh yes, that's right, I'll move it there.